src/fflogs.ts

```diff
@@ -0,0 +1,228 @@
+import {AxiosRequestConfig} from 'axios'
+
+// -----
+// Fight
+// -----
+
+export interface Fight {
+	id: number
+
+	boss: number
+	name: string
+	zoneID: number
+	zoneName: string
+	lastPhaseForPercentageDisplay?: number
+
+	difficulty?: number
+	bossPercentage?: number
+	fightPercentage?: number
+	kill?: boolean
+	partial?: number
+	size?: number
+	standardComposition?: boolean
+
+	start_time: number
+	end_time: number
```

Comment:
I like to annotate timestamps with a `/** */` doc comment to say in which format it is. I assume this is unix timestamp in seconds, so you'd write `/** Unix timestamp (s) */`.

It helps when you're also dealing with dates that might be ISO-8601 or some other format.